test/kb_quast_server_test.py

```diff
@@ -292,15 +289,15 @@ def test_fail_bad_quast_input_garbage(self):
         self.fail_quast(
             {'files': [{'path': 'data/greengenes_UnAligSeq24606.fa'},
                        {'path': 'data/greengenes_UnAligSeq24606_garbage.fa'}]},
-            'QUAST skipped some files - 2 expected, 1 processed.')
+            "'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte")
```

Comment:
This means this test isn't testing the conditions it's supposed to test. It should be modified so that it tests the case where QUAST skips a file.

Reply:
Hmm, I think failing early is a plus but I guess I can try to tweak the garbage data

Reply:
Both this test and the skip test are probably valuable